pipelines.adoc

```diff
@@ -1,266 +1,189 @@
+
+For more information, see link:https://github.com/kabanero-io/kabanero-pipelines[the kabanero-pipelines repo].
 
 == Running the pipelines
 
-=== Prereqs
-
-You have the Kabanero foundation installed on Red Hat Origin Community
-Distribution of Kubernetes (OKD) or OpenShift Container Platform (OCP)
-cluster. It has the necessary Kabanero, Isito, Knative, and Tekton
-components installed. Please refer to
-https://github.com/kabanero-io/kabanero-foundation for more details on
-installing the Kabanero foundation.
-
-Identify the Tekton Dashboard URL after your have completed the
-installation. You can login to your OKD cluster and run
-`+oc get routes+` to find this or check in the OKD console. This is
-useful for a few of the steps documented below. You can find more
-details about the dashboard at https://github.com/tektoncd/dashboard.
-
-NOTE: By default the pipelines will run in `+kabanero+` namespace and
-deploy the application there as well. If you would like to deploy the
-application in a different namespace, please update the app-deploy.yaml
-to point to the namespace to deploy to.
-
-==== Setup dynamic volume provisioning or a persistent volume
-
-Tekton pipelines requires a volume to be configured that will be used by
-the framework to share data across tasks. The build task, which
-leverages buildah, also requires a volume mount. The minimum required
-volume size is 5 Gi.
-
-If your cluster is running in a public cloud, dynamic volume
-provisioning is the easier and preferred option. Please refer to the
-following doc that describes how to point to the storage class using a
-config map.
-https://github.com/tektoncd/pipeline/blob/master/docs/install.md#how-are-resources-shared-between-tasks
-
-Note: If this is not configured and based on how your cloud provider’s
-default storage class is configred, your pipelines might create a new
-volume for each run which will increase your pipeline run execution
-time.
-
-Alternatively, if you are using an unmanaged cluster you can setup a
-persistent volume to be used by the pipeline. A simple example pv
-definition is provided. Update path and other values in pv.yaml to suit
-your requirements. For example, NFS is the preferred configuration in a
-production environment verus using local storage as shown in the
-example.
-
-Login to your cluster. For example for OKD,
-
-....
+=== Prerequisites
+
+. link:https://github.com/kabanero-io/kabanero-foundation[Kabanero foundation] must be installed on Red Hat Origin Community Distribution of Kubernetes (OKD) or the OpenShift Container Platform (OCP) cluster.
+
+. link:https://github.com/tektoncd/dashboard[Tekton Dashboard] is installed by default with Kabanero Foundation. To find the Tekton Dashboard URL, login to your OKD cluster and run `+oc get routes+` or check in the OKD console.
```

Comment:
Just found out they can also find the dashboard link from the kabanero landing page in the okd / ocp console.  So maybe we should add that here.